### PR TITLE
apply code style

### PR DESCRIPTION
**Pull request by LeftofZen**
more files changed with the new code style

### CODE REVIEW

File: src/OpenLoco/StationManager.cpp [hunk 1]
```diff
@@ -53,8 +53,8 @@ namespace OpenLoco::StationManager
     {
         if (Game::hasFlags(1u << 0) && !isEditorMode())
         {
-            const auto id = StationId(scenarioTicks() & 0x3FF);
-            auto station = get(id);
+            const auto kId = StationId(scenarioTicks() & 0x3FF);
```
Comment (duncanspumpkin) [inline src/OpenLoco/StationManager.cpp:56]: Although this is constant it's not a constant known at compile time so i don't think it should have the k prefix

File: src/OpenLoco/Ui/WindowManager.cpp [hunk 2]
```diff
@@ -35,7 +35,7 @@ namespace OpenLoco::Ui::WindowManager
 
     constexpr size_t max_windows = 12;
 
-    static loco_global<uint16_t, 0x0050C19C> time_since_last_tick;
+    static loco_global<uint16_t, 0x0050C19C> timeSinceLastTick;
```
Comment (duncanspumpkin) [inline src/OpenLoco/Ui/WindowManager.cpp:38]: Prefix with _

**Review by duncanspumpkin — COMMENTED** [review]
Looks good. Two minor corrections.

File: src/OpenLoco/StationManager.cpp (hunk 1 above)
Comment (LeftofZen) [reply src/OpenLoco/StationManager.cpp:56]: done

File: src/OpenLoco/Ui/WindowManager.cpp (hunk 2 above)
Comment (LeftofZen) [reply src/OpenLoco/Ui/WindowManager.cpp:38]: apologies for missing this. I also noticed a missing underscore on `gCurrentRotation` a few lines down from here, I added it there too

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**